src/test/java/com/google/javascript/clutz/DeclarationGeneratorTests.java

```diff
@@ -60,6 +69,15 @@ static File getGoldenFile(final File input) {
     return new File(input.getPath().replaceAll("\\.js$", ".d.ts"));
   }
 
+  static String maybeExternFileName(String testFileName) {
```

Comment:
I use `maybeDoSomething` for stuff that maybe does some side effect, and `getBlerghOrNull` for stuff that gets something or returns `null` if not found. YMMV, but I've found that quite useful.